### PR TITLE
fix error message in phone

### PR DESCRIPTION
**Pull request by yedidya03**

### CODE REVIEW

File: packages/donor/src/screens/extendedSignup/PersonalDetails.tsx [hunk 1]
```diff
@@ -43,8 +43,17 @@ export default function PersonalDetails(props: PersonalDetailsProps) {
     undefined
   );
 
+  const isPhoneNumberAllNumbers = /^[0-9]*$/.test(phone);
   const phoneValidator = /^05(?!6)\d{8}$/;
   const isValidPhone = phoneValidator.test(phone);
+
+  let phoneMessage = undefined;
```
Comment (yaron1m) [inline packages/donor/src/screens/extendedSignup/PersonalDetails.tsx:50]: errorMessage

File: packages/donor/src/screens/extendedSignup/PersonalDetails.tsx [hunk 2]
```diff
@@ -43,8 +43,17 @@ export default function PersonalDetails(props: PersonalDetailsProps) {
...
+  if (!isPhoneNumberAllNumbers) {
+    phoneMessage = "יש להזין ספרות בלבד";
+  } else if (phone.length > 0 && !isValidPhone) {
+    phoneMessage = "מספר הטלפון אינו חוקי";
```
Comment (yaron1m) [inline packages/donor/src/screens/extendedSignup/PersonalDetails.tsx:54]: אינו תקין

**Review by yaron1m — APPROVED** [review]

File: packages/donor/src/screens/extendedSignup/PersonalDetails.tsx (hunk 2 above)
Comment (yedidya03) [reply packages/donor/src/screens/extendedSignup/PersonalDetails.tsx]: fixed.

File: packages/donor/src/screens/extendedSignup/PersonalDetails.tsx (hunk 1 above)
Comment (yedidya03) [reply packages/donor/src/screens/extendedSignup/PersonalDetails.tsx:50]: didn't understand (it stayed like before - error message or undefined if no error)

Comment (yaron1m) [reply packages/donor/src/screens/extendedSignup/PersonalDetails.tsx:50]: I meant change the variable name to `errorMessage`, not `phoneMessage`

Comment (yedidya03) [reply packages/donor/src/screens/extendedSignup/PersonalDetails.tsx]: Oh, ok.

**Comment by yaron1m** [comment]
Try running `yarn install` from the root folder and then run style

**Pull request merged**